public/components/datasources/components/manage/data_connection.tsx

```diff
@@ -165,6 +166,35 @@ export const DataConnection = (props: any) => {
         />
       ),
     },
+    {
+      id: 'associated_objects',
+      name: 'Associated Objects',
+      disabled: false,
+      content: (
+        <AssociatedObjectsTab
+          associatedObjects={[
+            {
+              id: '1',
```

Comment:
@ps48 do you want me to remove this sample data? Or how are we gonna test this?

Reply:
Let's keep the sample data here.